wd_core.au3

```diff
@@ -1397,15 +1350,8 @@ Func __WD_Get($sURL)
 		$iResult = $_WD_ERROR_InvalidValue
 	EndIf
 
-	If $_WD_DEBUG = $_WD_DEBUG_Info Then
-		__WD_ConsoleWrite($sFuncName & ': StatusCode=' & $_WD_HTTPRESULT & "; $iResult = " & $iResult & "; $sResponseText=" & StringLeft($sResponseText, $_WD_RESPONSE_TRIM) & "...")
-	EndIf
-
-	If $iResult Then
-		Return SetError(__WD_Error($sFuncName, $iResult, $sResponseText), $_WD_HTTPRESULT, $sResponseText)
-	EndIf
-
-	Return SetError($_WD_ERROR_Success, 0, $sResponseText)
+	Local $sMessage = (($sResponseText) ? ("ResponseText=" & StringLeft($sResponseText, $_WD_RESPONSE_TRIM) & "...") : (""))
+	Return SetError(__WD_Error($sFuncName, $iResult, $sMessage), $_WD_HTTPRESULT, $sResponseText)

```

Comment:
Are we still supposed to be returning the HTTP status in @extended in __WD_Get/Put/Delete?

Reply:
It is only used in `__WD_Get/Post/Delete` and they should be used by user/scripter.
So we only must check if any other functicion from core/helper UDF which use `__WD_Get/Post/Delete` relays on `@extended`.

Reply:
why there is `See w3c/webdriver@35df53a for details.` this is not link.

Reply:
It's a commit in the w3c/webdriver repo. The full link is https://github.com/w3c/webdriver/commit/35df53a8a71500e264497f0cbda9f30a94e548cc

Reply:
> It is only used in that way in `__WD_Get/Post/Delete` and they should be used by user/scripter. So we only must check if any other functicion from core/helper UDF which use `__WD_Get/Post/Delete` relays on `@extended`.

I checked it and it looks like there is no one function that uses `@exteneded` value returned after calling `__WD_Get/Post/Delete` 

for example `_WD_Windows` check for `$_WD_HTTPRESULT`
```auotit
	If $iErr = $_WD_ERROR_Success Then
		If $_WD_HTTPRESULT = $HTTP_STATUS_OK Then
```

but `$_WD_HTTPRESULT` is set internally by  `__WD_Get/Post/Delete` 

So I think that we can remove  `__WD_Get/Post/Delete`  from `SetError()` return points in this 3 functions:  `__WD_Get/Post/Delete`